ophyd/v2/tests/test_core.py

```diff
@@ -1,10 +1,39 @@
 import asyncio
 import re
+from typing import Dict, Union
 
 import pytest
 from bluesky.protocols import Status
+from bluesky.run_engine import RunEngine, TransitionError
 
-from ophyd.v2.core import AsyncStatus, Signal
+from ophyd.v2.core import (
+    AsyncStatus,
+    Device,
+    DeviceDict,
+    Signal,
+    StandardReadable,
+    get_device_children,
+)
+
+
+@pytest.fixture(scope="function", params=[False, True])
+def RE(request):
+    loop = asyncio.new_event_loop()
+    loop.set_debug(True)
+    RE = RunEngine({}, call_returns_result=request.param, loop=loop)
```

Comment:
Why do we want to test it with `call_returns_result` true and false?

Reply:
Wait, do we even use this?

Reply:
no - will remove.